toadr3/event.py

```diff
@@ -48,8 +48,9 @@ def __init__(self, data: dict):
         if "priority" in data:
             try:
                 self._priority = int(data["priority"])
-            except ValueError:
-                raise SchemaException("Expected 'priority' to be an integer in event schema.")
+            except ValueError as err:
+                msg = "Expected 'priority' to be an integer in event schema."
+                raise SchemaException(msg) from err
```

Comment:
Why is this better?

Reply:
It will add the original ValueError as the cause of the SchemaException. 

[Like this](https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
[Explanation](https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/)

In this case, I thought providing the ValueError would only be helpful.

Reply:
I meant why it was good to have 'msg' vs inlining the string. Line length?

Reply:
Ah, yes. Line length. :)